pkg/apiserver/certificate/certificate_test.go

```diff
@@ -161,6 +184,18 @@ func TestApplyServerCert(t *testing.T) {
 				}
 				return
 			}
+
+			if tt.selfSignedCert {
+				oldCertKeyContent := got.getCertificate()
+				time.Sleep(8 * time.Second)
+				newCertKeyContent := got.getCertificate()
+				if tt.testRotate {
+					assert.NotEqual(t, oldCertKeyContent, newCertKeyContent, "CA Cert not updated")
+				} else {
+					assert.Equal(t, oldCertKeyContent, newCertKeyContent, "CA Cert updated")
+				}
```

Comment:
please use `wait.Poll` here (https://godoc.org/k8s.io/apimachinery/pkg/util/wait#Poll)

there is no need to sleep for 8 seconds if you don't need to, but you can use 8 seconds as your timeout